lib/loga.rb

```diff
@@ -8,22 +8,25 @@
 require 'loga/rack/request'
 require 'loga/rack/request_id'
 require 'loga/railtie' if defined?(Rails)
+require 'loga/log_subscribers/action_mailer' if defined?(ActiveSupport::LogSubscriber)
```

Comment:
Any particular reason why this isn't in the `Railtie`? Presumably non Rails that make use of `ActiveSupport::LogSubscriber`?

Reply:
@timstott, yes, this is the intention of requiring `Loga::LogSubscribers::ActionMailer` here. I believe, someone might want to use ActiveSupport and Loga in a non Rails app. Does this mess up something in the project?

Reply:
Let's move this in the railtie, it won't be hooked if there isn't a rails app.